File: java/konstanta.java
import java.util.Scanner;
public class konstanta {
    public static void main(String[] args) {
        final double PI = 3.14159;
        @SuppressWarnings("resource")
        Scanner sc = new Scanner(System.in);
        System.out.print("Masukkan jari-jari: ");
        double r = sc.nextDouble();

        double luas = PI * r * r;
        System.out.println("Luas Lingkaran = " + luas);
    }
}

File: c++/konstanta.cpp
#include <iostream>
using namespace std;

int mani(){
    const float PI = 3.14159;
    float r;
    cout << "masukan jari-jari: ";
    cin >>r;
    float luas =PI * r * r;
    cout << "Luas Lingkaran = " << luas;
    return 0;

}